addons/disposable/XEH_preInit.sqf

```diff
@@ -27,6 +27,7 @@ GVAR(NormalLaunchers) = [] call CBA_fnc_createNamespace;
 GVAR(LoadedLaunchers) = [] call CBA_fnc_createNamespace;
 GVAR(UsedLaunchers) = [] call CBA_fnc_createNamespace;
 GVAR(magazines) = [];
+GVAR(BackpackReplaceableMagazines) = [];
```

Comment:
```suggestion
GVAR(BackpackLaunchers) = createHashMap;
```

Reply:
Should i ~~break~~ change other lists/namespaces into hashMaps too?